config/examples/delta/Anycubic/Predator/Configuration.h

```diff
@@ -2396,6 +2396,7 @@
 // 480x320, 3.5", FSMC Stock Display from AnyCubic
 //
 #define ANYCUBIC_TFT35
+#define TFT_ROTATION TFT_ROTATE_180
```

Comment:
There's already a `TFT_ROTATION` define further down in `Configuration.h`:

https://github.com/MarlinFirmware/Configurations/blob/8e78071ee2a69ab87734431feec4915f6978140c/config/examples/delta/Anycubic/Predator/Configuration.h#L2446